src/common/links.ts

```diff
@@ -51,3 +52,30 @@ export function isValidHttpUrl(link: string): boolean {
     return false;
   }
 }
+
+export const getShortUrl = async (url: string | URL): Promise<string> => {
+  const urlSHortenerSecret = process.env.URL_SHORTENER_SECRET;
+
+  if (!urlSHortenerSecret) {
+    return url.toString();
+  }
```

Comment:
It would be great if we could add some kind of runtime check if all env variables are present before starting the app in production. It could run in CI/CD. Because if we were to run the app in production we could be serving not shortened URLs if we forget to add `URL_SHORTENER_SECRET`. Similar thing already exists with `CLOUDINARY_URL`.

Just wanted to see if we looked at this before or have any ideas?

Reply:
Valid point. Let's log it and think how to approach later

Reply:
Logged here https://dailydotdev.atlassian.net/browse/WT-1428